### ISSUE TITLE
val_dataloader problem

### ISSUE THREAD

**Issue by AceSix** [parksunwoo/show_attend_and_tell_pytorch#3]
how can train loader return 3 items and val loader return 4 items while they use the same function?

**Comment by parksunwoo** [comment]
You are right. dataloader will return (images, captions, lengths) for every iteration..
I think I'll need to fix it. Thanks

**Comment by dreamyuchen** [comment]
> You are right. dataloader will return (images, captions, lengths) for every iteration..
> I think I'll need to fix it. Thanks

Have you fixed this problem? 
How should the code be modified?look forward to your reply.
